common/DSUtilLite/growarray.h

```diff
@@ -32,6 +32,10 @@
 #include <assert.h>
 #include "DShowUtil.h"
 
+#ifndef DWORD_MAX
+#define DWORD_MAX 4294967295
```

Comment:
I'm so sorry, I don't know how I missed that bit of information when I looked up the DWORD max value. Yes, including IntSafe.h is obviously the right way to do this.